draft-privacypass-rate-limit-tokens.md

```diff
@@ -120,14 +121,113 @@ periodically, say, once per month. For example, an origin may serve ten (major
 content) requests in a month before a paywall is enacted. Origins may want to
 differentiate quick refreshes from distinct accesses.
 
-For some applications, the basic issuance protocol from {{?BASIC-ISSUANCE=I-D.ietf-privacypass-protocol}}
-could be used to implement rate limits. In particular, the 'Joint Attester and Issuer' model from {{ARCH}} could
-be used to restrict the number of tokens issued to individual clients over a
+For some applications, the basic issuance protocol from {{ISSUANCE}} could be used to
+implement rate limits. In particular, the 'Joint Attester and Issuer' model from {{ARCH}}
+could be used to restrict the number of tokens issued to individual clients over a
 time window. However, in this deployment model, the Attester and Issuer would learn all
-origins used by a participating client. In some cases this might be a significant portion of
-browsing history. The issuance protocol defined in this document employs the 'Split Origin, Attester, Issuer'
-model to combat this, where the issuer would know all per-origin policies, and the
-attester would mantain per-client state without knowing all origins a client visits.
+origins used by a specific client, thereby coupling sensitive attestation and redemption
+contexts. In some cases this might be a significant portion of browsing history.
+
+## Protocol Overview
+
+The issuance protocol defined in this document decouples sensitive attestation context
+such as the client identity from the redemption context such as the origin. It does
+so by employing the 'Split Origin, Attester, Issuer' model. In this model, the issuer
+learns redemption information like per-origin rate limit policies, and the attester
+learns attestation information like client identity used to maintain and enforce
+rate limits based on encrypted origins.
+
+{{fig-example-allow}} shows how this interaction works for client requests that
+are within the rate limit. The client's token request to the attester (constructed
+according to {{request-one}}, and forwarded to the issuer according to {{request-two}})
+contains encrypted information that the issuer uses to identify the relevant rate
+limit policy to apply. This rate limit policy is returned to the attester (according
+to {{response-one}}), which then checks whether or not the client is within this
+policy. If yes, the attester forwards the issuer token response to the client (according
+to {{response-two}}) so that the resulting token can be redeemed to the origin.
```

Comment:
```suggestion
are within the rate limit. The Client's token request to the Attester (constructed
according to {{request-one}}, and forwarded to the Issuer according to {{request-two}})
contains encrypted information that the Issuer uses to identify the relevant rate
limit policy to apply. This rate limit policy is returned to the Attester (according
to {{response-one}}), which then checks whether or not the Client is within this
policy. If yes, the Attester forwards the issuer token response to the Client (according
to {{response-two}}) so that the resulting token can be redeemed by the Origin.
```